template/Dockerfile/images/hhvm.jinja2

```diff
@@ -12,7 +12,8 @@
         graphicsmagick \
         ghostscript \
     && /usr/bin/update-alternatives --install /usr/bin/php php /usr/bin/hhvm 60 \
-    && curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin/ --filename=composer
+    && curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin/ --filename=composer \
+    && curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin/ --filename=composer1 --version=1.10.16
```

Comment:
can we not pin composer version 1?
```suggestion
    && curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin/ --filename=composer1 --version=1.10.16
    && composer1 self-update --1
```
info: https://github.com/composer/composer/releases/tag/2.0.0-RC2

Reply:
what benefit does your suggestion have?

Reply:
We always have the newest composer 1.* version available

Reply:
For that, the composer-setup.php script provides the channel parameter, that basically does the same:
```suggestion
    && curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin/ --filename=composer1 --1
```